apps/web/scripts/start.js

```diff
@@ -81,4 +84,17 @@ async function start () {
   }
 }
 
+function getLocalIP () {
```

Comment:
what is all this???? And what does it have to do with mobile-web improvements?

Reply:
For testing on a phone over a local network

Reply:
Maybe you were able to connect with your phone, but I was not able to before. The way it was before, it was only accepting connections from localhost. Perhaps I did this in the wrong way?

Reply:
Oh, so a physical device??

I have the emulator/stimulator setup, so if you are somehow connecting with a physical device, then I wouldn't know what extra config is needed. Out of curiosity, are you plugged in via cable or is it connecting over the network?

Reply:
oh interesting! Could we document this in one of the READMEs?